contracts/src/bin/deploy.rs

```diff
@@ -0,0 +1,120 @@
+//! Script to deploy Gnosis Protocol contracts to a local test network.
+//! Additionally writes the deployed addresses to the `target` directory so that
+//! they can be used by the build script.
```

Comment:
Maybe there should be a short readme in the contracts folder that explains on a high level what we do in this crate and why and which paths are used.